CONTRIBUTING.md

```diff
@@ -111,16 +111,34 @@ $ meson test
 
 ## Windows support (experimental)
 
-To run on windows, install [msys2](msys2.org) and follow the instructions to set up a development environment. Once the setup is done, install the following dependencies:
+To run on windows, install [msys2](msys2.org) and follow the instructions to set up a development environment. Once the configuration is done, follow the instructions:
+
+### Install the dependencies
 
 ```bash
 $ pacman -Suy
-$ pacman -S git mingw-w64-ucrt-x86_64-meson mingw-w64-ucrt-x86_64-gtk4 mingw-w64-ucrt-x86_64-python3 mingw-w64-ucrt-x86_64-python3-gobject mingw-w64-ucrt-x86_64-libadwaita mingw-w64-ucrt-x86_64-gstreamer mingw-w64-ucrt-x86_64-gst-plugins-good mingw-w64-ucrt-x86_64-gtksourceview5 mingw-w64-ucrt-x86_64-librsvg mingw-w64-ucrt-x86_64-desktop-file-utils
+$ pacman -S git mingw-w64-ucrt-x86_64-meson mingw-w64-ucrt-x86_64-gtk4 mingw-w64-ucrt-x86_64-python3 mingw-w64-ucrt-x86_64-python3-gobject mingw-w64-ucrt-x86_64-libadwaita mingw-w64-ucrt-x86_64-gstreamer mingw-w64-ucrt-x86_64-gst-plugins-good mingw-w64-ucrt-x86_64-gtksourceview5 mingw-w64-ucrt-x86_64-librsvg mingw-w64-ucrt-x86_64-desktop-file-utils unzip meson cmake
+```
+
+### Install pandoc
+
+Now we download pandoc and copy it to `/usr/bin` by entering the msys2 terminal and typing the following commands
+```bash
+$ cd /tmp # Go to temporary directory
+$ wget https://github.com/jgm/pandoc/releases/download/3.1.13/pandoc-3.1.13-windows-x86_64.zip # Download pandoc zip
+$ unzip pandoc-3.1.13-windows-x86_64.zip # Unzip it
+$ cp pandoc-3.1.13/pandoc.exe /usr/bin # Copy it
```

Comment:
You could simplify this to something like

```bash
$ wget https://github.com/jgm/pandoc/releases/download/3.1.13/pandoc-3.1.13-windows-x86_64.zip -P /tmp
$ unzip -j /tmp/pandoc-3.1.13-windows-x86_64.zip pandoc-3.1.13/pandoc.exe -d /usr/bin/
```

Reply:
Maybe this is a github bug, but I see this as _resolved_, but the code hasn't changed.